sql/item_latents.sql

```diff
@@ -686,18 +682,22 @@ INSERT INTO `item_latents` VALUES(16238, 369, 1, 13, 3);
 INSERT INTO `item_latents` VALUES(14954, 5, 35, 14, 0);
 INSERT INTO `item_latents` VALUES(14954, 168, -5, 14, 0);
 INSERT INTO `item_latents` VALUES(14954, 71, 1, 14, 0);
-INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 2);
+
+-- -------------------------------------------------------
+-- Sirius Axe
+-- -------------------------------------------------------
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 3);
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 4);
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 5);
+INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 6);
 
 -- -------------------------------------------------------
 -- Company Sword
 -- -------------------------------------------------------
-INSERT INTO `item_latents` VALUES(17662, 366, 2, 15, 2); -- DMG 46~54, increases by 2 for every person in party (excl. wearer)
```

Comment:
crap. it does cap out at 54 in every source. somethings off. somebody failed at math somewhere and its been repeated since.

http://ffxiclopedia.wikia.com/wiki/Company_Sword
```
Damage on Company Sword increases by 2 for each party member over 2, up to 54 for a party of 6 people.
```
https://www.bg-wiki.com/bg/Company_Sword
```
This weapon's DMG rating increases by 2 for each party member in the player's party excluding themself.
```
http://www.ffxiah.com/item/17662/company-sword
```Damage on Company Sword increases by 2 for each party member over 2, up to 54 (DPS: 12.27) for a party of 6 people.
```